data_steward/tools/consolidated_achilles_report/get_all_results_html.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import gcs_utils
+import json
+import os
+import query_reports
+
+DRC_BUCKET_PATH = 'gs://%s/' % gcs_utils.get_drc_bucket()
+
+
+def get_hpo_id(p):
+    rel_path = p[len(DRC_BUCKET_PATH):]
+    return rel_path[:rel_path.index('/')]
+
+
+def transform_bq_list(uploads):
+    """
+    Get paths to all most recent report files
+
+    :return:
+    """
+    results = []
+    for upload in uploads:
+        dte, p = upload['upload_timestamp'], upload['file_path']
+        hpo_id = get_hpo_id(p)
+        report_path = p.replace('person.csv', 'results.html')
+        result = dict(hpo_id=hpo_id, updated=dte, report_path=report_path)
```

Comment:
I haven't used this method of setting up a dictionary before.  It is elegant.  There is a small performance penalty, but it likely won't be noticed in this case.  I tested it using:
```
import timeit
timeit.timeit("a = dict(alpha='alphabetic', beta='betty', zeta='zebedee')", number=1000000)
timeit.timeit("a = {'alpha': 'alphabetic', 'beta': 'betty', 'zeta': 'zebedee'}", number=1000000)
```
The second method ran in a little less than half the same amount of time as the first.